description: |
  ### Impact
  If you run teler inside a Docker container and encounter `errors.Exit` function, it will cause denial-of-service (`SIGSEGV`) because it doesn't get process ID and process group ID of teler properly to kills.

  ### Patches
  Upgrade to the >= 0.0.1 version.

  ### Workarounds
  N/A

  ### References
  - https://github.com/kitabisa/teler/commit/ec6082049dba9e44a21f35fb7b123d42ce1a1a7e

  ### For more information
  If you have any questions or comments about this advisory:
  * Open an issue in [Issues Section](https://github.com/kitabisa/teler/issues)
  * Email us at [infosec@kitabisa.com](mailto:infosec@kitabisa.com)
ghsaId: GHSA-jhj6-5mh6-4pvf
identifiers:
- type: GHSA
  value: GHSA-jhj6-5mh6-4pvf
- type: CVE
  value: CVE-2020-26213
permalink: https://github.com/advisories/GHSA-jhj6-5mh6-4pvf
publishedAt: "2021-05-24T17:00:46Z"
references:
- url: https://github.com/kitabisa/teler/security/advisories/GHSA-jhj6-5mh6-4pvf
- url: https://nvd.nist.gov/vuln/detail/CVE-2020-26213
- url: https://github.com/advisories/GHSA-jhj6-5mh6-4pvf
summary: Denial-of-Service within Docker container
updatedAt: "2021-10-11T21:06:20Z"
vulnerabilities:
- firstPatchedVersion:
    identifier: 0.0.1
  package:
    name: ktbs.dev/teler/pkg/errors
  severity: MODERATE
  updatedAt: "2021-05-21T22:09:37Z"
  vulnerableVersionRange: < 0.0.1
